llvm/test/CodeGen/SystemZ/rot-02.ll

```diff
@@ -2,7 +2,8 @@
 ; Test removal of AND operations that don't affect last 6 bits of rotate amount
 ; operand.
 ;
-; RUN: llc < %s -mtriple=s390x-linux-gnu | FileCheck %s
+; RUN: llc < %s -mtriple=s390x-linux-gnu | FileCheck %s -check-prefixes=CHECK,CHECK-LV
+; RUN: llc < %s -mtriple=s390x-linux-gnu -early-live-intervals | FileCheck %s -check-prefixes=CHECK,CHECK-LIS
```

Comment:
These changes look redundant?

Reply:
Ugh. I am still struggling with how to deal with stacked patches. In the first commit the separate prefixes are necessary. In the second patch, codegen improves and they are no longer needed.